source/team/team.html.md.erb

```diff
@@ -17,4 +17,4 @@ We are currently made up of 9 members:
 - Stephen Linden, WebOps Engineer
 - Jake Mulley, Head of Hosting and Digital Operations
 - Sean Privett, Senior Product Manager
```

Comment:
Shouldn't this read `Shakeel Quader` as Sean has moved on to a different position?

Reply:
@dms1981 that change is in https://github.com/ministryofjustice/modernisation-platform/pull/1939.